benchmark/profile_throughput.py

```diff
@@ -136,8 +146,8 @@ def process_request(self,
         stats = []
         while not res_queue.empty():
             session_id, _stats = res_queue.get()
-            print(f'\n{"-" * 50}\n'
-                  f'session {session_id} stats: \n{_stats}\n{"-" * 50}\n')
+            # print(f'\n{"-" * 50}\n'
```

Comment:
do we need to remove these two lines?

Reply:
It's very long if `num_prompts` is large. Any sepcial reason to keep it?